Packs/USTA/Integrations/USTA/USTA.py

```diff
@@ -0,0 +1,656 @@
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs=referredResult
+    )
+
+
+def search_specific_identity_leaks() -> CommandResults:
+
+    url = demisto.params().get('url')[:-1] if str(demisto.params().get('url')).endswith('/') \
+        else demisto.params().get('url')
+    CREDENTIALS = demisto.params().get('apikey')
+    # AUTH_HEADERS = {'Content-Type': 'application/json'}
+
+    params = "TOKEN " + CREDENTIALS
+    headers = {"Authorization": params,
+               "Content-Type": "application/json"
+               }
+
+    base_url = urljoin(url, '/api')
+    endpoint = "/threat-stream/identity-leaks"
+
+    url = urljoin(base_url, endpoint)
+
+    identities = demisto.args().get('identity_number')
+    identityList = identities.split(",")
+
+    hashedIdentities = encodeData(identityList)
+
+    mappingHashIdentity = []
+    for i in range(len(identityList)):
+        mappingDict = {
+            'Identity': identityList[i],
+            'Identity Hash': hashedIdentities[i]
+        }
+        mappingHashIdentity.append(mappingDict)
+
+    data = {
+        'signatures': hashedIdentities
+    }
+
+    req = requests.patch(url, headers=headers, data=json.dumps(data), verify=VERIFY_SSL)
+    identityLeaks = json.loads(req.content)
+
+    readable_output = tableToMarkdown('Identity Leaks', identityLeaks)
+    readable_output += tableToMarkdown('Identity Hash Mapping', mappingHashIdentity)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs=identityLeaks
+    )
+
+
+def close_incident() -> CommandResults:
+
+    url = demisto.params().get('url')[:-1] if str(demisto.params().get('url')).endswith('/') \
+        else demisto.params().get('url')
+    CREDENTIALS = demisto.params().get('apikey')
+    # AUTH_HEADERS = {'Content-Type': 'application/json'}
+
+    params = "TOKEN " + CREDENTIALS
+    headers = {"Authorization": params,
+               "Content-Type": "application/json"
+               }
+
+    base_url = urljoin(url, '/api')
+    endpoint = "/threat-stream/close-ticket"
+
+    url = urljoin(base_url, endpoint)
+
+    inc_id = demisto.args().get('id')
+    data = {
+        'id': inc_id
+    }
+
+    req = requests.patch(url, headers=headers, data=json.dumps(data), verify=VERIFY_SSL)
+    closedIncident = json.loads(req.content)
+
+    readable_output = tableToMarkdown('Close Incident', closedIncident)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs=closedIncident
+    )
+
+
+def encodeData(identities: list) -> list:
+    hashedList = []
+    for identity in identities:
+        hashedIdentity = hashlib.sha256((hashlib.md5(identity.encode())).hexdigest().encode()).hexdigest()
+        hashedList.append(hashedIdentity)
+    return hashedList
+
+
+def timeToEpoch(time: str):
+    timeParameterList = time.split("-")
+    if len(timeParameterList) != 5:
+        error_message = "Time paremeter is wrong"
```

Comment:
There is no time parameter. did you mean argument?

Reply:
Time parameter is taken from functions. For example: search_malicious_urls function call with  
if start:
        startDate = timeToEpoch(start)

Reply:
So this called an argument, please change your error message.
Parameters are used for configuring the integration

Reply:
Ok, I missed this, fixed it, thanks :)